src/libcore/sync/atomic.rs

```diff
@@ -926,10 +926,24 @@ macro_rules! atomic_int {
      $stable_cxchg:meta,
      $stable_debug:meta,
      $stable_access:meta,
+     $s_int_type:expr, $int_ref:expr,
      $int_type:ident $atomic_type:ident $atomic_init:ident) => {
         /// An integer type which can be safely shared between threads.
         ///
-        /// This type has the same in-memory representation as the underlying integer type.
+        /// This type has the same in-memory representation as the underlying
+        /// integer type, [`
+        #[doc = $s_int_type]
+        /// `](
+        #[doc = $int_ref]
+        /// ). For more about the differences between atomic types and
+        /// non-atomic types, please see the [module-level documentation].
+        ///
+        /// Please note that examples are shared between atomic variants of
+        /// primitive integer types, so it's normal that they are all
+        /// demonstrating [`AtomicIsize`].
```

Comment:
... this?

Reply:
You mean to replace that particular item? All of the examples are actually using `AtomicIsize`, so that's exactly the atomic int type that should be there.

I thought about fixing (meta-parameterizing) the examples, but that is unfortunately _not_ doable because of the way the docs are generated (I tried!).

Reply:
I meant the examples. No worries, I didn't think it was actually possible, but I didn't realize the above was possible either, so... 😄 👍